time-based-packet-transmission-offload.md

```diff
@@ -95,16 +98,21 @@ if(setsockopt(sock, SOL_SOCKET, SO_TXTIME_SCHED, (char*)&optVal, optLen) == SOCK
 
 ## Sending Packets
 The app calls `WSASendMsg`.   
-The app passes ancillary data in the form of `TPTO_ANCILLARY_DATA`
+The app passes ancillary data in the form of `TPTO_ANCILLARY_DATA`.
 ```C
 typedef struct _TPTO_ANCILLARY_DATA {
     uint64_t TxTime;
-    uint8_t  DropIfRate
+    uint64_t TimeDelta;
 } TPTO_ANCILLARY_DATA;
 ```
 
 - TxTime to indicate when the associated packet is sent
-- DropIfRate to indicate whether to drop packet if it cannot be transmitted by the given deadline
+- TimeDelta is an interval to transmit segmented packets by LSO/USO
```

Comment:
The updated text still doesn't make it clear what this means. You need text like "time by which packets in an LSO send should be spread out". (But also, that's not the plan, right? It seems like small bursts of a few packets should be alright and more efficient.)

Reply:
I and Nick discussed yesterday. It is also mentioned in Issue
https://github.com/microsoft/quic-offloads/issues/49

Reply:
```suggestion
- TimeDelta is an interval, starting at `TxTime`, to transmit individual packets that have been coalesced into a single send (by LSO/USO)
```